persistence/src/test/java/edu/unc/lib/dl/persist/services/destroy/DestroyObjectsJobIT.java

```diff
@@ -143,14 +145,17 @@ public void destroyObjectsInSameTreeTest() {
         PID folderObjPid = objsToDestroy.get(0);
         verify(spyProxyService).destroyProxy(folderObjPid);
 
-        FileObject fileObj = repoObjLoader.getFileObject(fileObjPid);
-        WorkObject workObj = repoObjLoader.getWorkObject(workObjPid);
-        FolderObject folderObj = repoObjLoader.getFolderObject(folderObjPid);
-        assertTrue(fileObj.getModel().contains(fileObj.getResource(), RDF.type, Cdr.Tombstone));
-        assertTrue(workObj.getModel().contains(workObj.getResource(), RDF.type, Cdr.Tombstone));
-        assertTrue(folderObj.getModel().contains(folderObj.getResource(), RDF.type, Cdr.Tombstone));
-
-        PremisEventObject event = repoObjLoader.getPremisEventObject(folderObj.getPremisLog().listEvents().get(0));
+        Tombstone stoneFile = repoObjLoader.getTombstone(fileObjPid);
+        Tombstone stoneWork = repoObjLoader.getTombstone(workObjPid);
+        Tombstone stoneFolder = repoObjLoader.getTombstone(folderObjPid);
+        assertTrue(stoneFile.getModel().contains(stoneFile.getResource(), RDF.type, Cdr.Tombstone));
```

Comment:
I'm okay if you want to keep this as is, but it may be slightly more readable to adjust these assertions to 
`assertTrue(stoneFile.getResource().hasProperty(RDF.type, Cdr.Tombstone));`

Reply:
sure, fixed!